pkg/apis/volumesnapshot/v1beta1/types.go

```diff
@@ -163,6 +169,9 @@ type VolumeSnapshotStatus struct {
 // VolumeSnapshotClasses are non-namespaced
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Cluster
+// +kubebuilder:printcolumn:name="Driver",type=string,JSONPath=`.driver`
+// +kubebuilder:printcolumn:name="DeletionPolicy",type=string,JSONPath=`.deletionPolicy`,description="Determines whether a VolumeSnapshotContent created through the VolumeSnapshotClass should be deleted when its bound VolumeSnapshot is deleted."
+// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
```

Comment:
s/Age/CreationTime

Reply:
I think the creation timestamp of a snapshot class does not need to be shown here.

Reply:
I've removed the creation time.